### ISSUE TITLE
Spastic triplegia

### ISSUE THREAD

**Issue by jagcsun** [obophenotype/human-phenotype-ontology#7315]
**Preferred term label:**
Spastic triplegia

**Synonyms**

**Definition (free text, please give PubMed ID)**
Spastic weakness  found in both lower extremities and one upper extremity.
OR
A type of cerebral palsy which is characterized by spastic weakness  found in both lower extremities and one upper extremity. 
(PMID: 32821558 and 27188686)

**Comment**
The lower limb is invariably more affected on the same side as the upper limb involvement. 

**Parent term (use hpo.jax.org/app)**
Spasticity HP:0001257
OR
Cerebral palsy HP:0100021

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-6670-9157

@pnrobinson

**Comment by peupeubangbang** [comment]
Hi @jagcsun suggest you link to both parent terms?

This would fit in with classification used by the European (SCPE) and Australian (ACPER) Cerebral Palsy Registers. I will add this comment for the other spastic CP types (diplegia, quadriplegia, hemiplegia). 
https://eu-rd-platform.jrc.ec.europa.eu/scpe/cerebral-palsy_en

**Comment by pnrobinson** [comment]
Adding term. Also added multiple parentage as suggested by @peupeubangbang 
thanks!
